app/src/main/java/com/x8bit/bitwarden/ui/autofill/fido2/manager/Fido2CompletionManagerImpl.kt

```diff
@@ -63,14 +62,6 @@ class Fido2CompletionManagerImpl(
         activity.also {
             val intent = Intent()
             when (result) {
-                Fido2CredentialAssertionResult.Cancelled -> {
```

Comment:
What's the reason for deleting this case?

Reply:
Great question. There is no opportunity for the assertion to be explicitly cancelled, short of backing out of the app, which responds to the OS with a cancellation signal. The user will only see Bitwarden while verification takes place. Cancelling or failing verification will display an error dialog to the user and complete the process by reporting an error to the OS. See the clip from #3630 below for an example.

![passkey-assertion-unlocked](https://github.com/user-attachments/assets/e15cbe3c-1aa5-4397-99a8-82f406c3ebc1)

Reply:
Very helpful, thank you!